sync/notify/wptfyimsg.py

```diff
@@ -156,7 +160,29 @@ def add_result(result, test_name, subtest_name):
     return details
 
 
-def value_str(result, head_browsers, base_browsers):
+def get_meta(metadata, test, subtest, status):
+    test_metadata = metadata.get(test)
+    if not test_metadata:
+        return []
+
+    specific_metadata = []
+    for item in test_metadata:
+        if "results" not in item:
+            specific_metadata.append(item)
+        else:
+            specific_results = [result for result in item["results"]
+                                if (("subtest" not in item or
+                                     item["subtest"] == subtest) and
+                                    ("status" not in item or
+                                     item["status"] == status))]
```

Comment:
This is very difficult to read, is there any way to make this easier to understand?

Reply:
Pushed something; idk if it's what you're looking for

Reply:
Looks much better to me, thanks!